lib/resources/datav2/entityv2.ts

```diff
@@ -357,7 +374,15 @@ export function AlpacaCryptoQuote(data: RawCryptoQuote): CryptoQuote {
 }
 
 export function AlpacaCryptoBar(data: RawCryptoBar): CryptoBar {
-  return aliasObjectKey(data, crypro_bar_mapping) as CryptoBar;
+  return aliasObjectKey(data, crypto_bar_mapping) as CryptoBar;
+}
+
+export function AlpacaCryptoSnapshot(data: any): CryptoSnapshot {
```

Comment:
You could create a `RawCryptoSnaphot` type if you want to, but not obligatory

Reply:
I didn't do this because `AlpacaSnapshot` doesn't have a raw interface other. I'm leaving it so that they are consistent for now... :thinking: